video/out/vo.c

```diff
@@ -1066,7 +1066,7 @@ static MP_THREAD_VOID vo_thread(void *ptr)
         vo->driver->control(vo, VOCTRL_CHECK_EVENTS, NULL);
         bool working = render_frame(vo);
         int64_t now = mp_time_ns();
-        int64_t wait_until = now + (working ? 0 : (int64_t)1e9);
+        int64_t wait_until = now + (working ? 0 : MP_TIME_S_TO_NS(1000));
```

Comment:
```suggestion
        int64_t wait_until = now + MP_TIME_S_TO_NS(working ? 0 : 1000);
```

Maybe it is only me, but looks more readable, feel free to ignore.

Reply:
Works for me too.